azure_monitor/src/azure_monitor/sdk/auto_collection/live_metrics/manager.py

```diff
@@ -0,0 +1,152 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+#
+import threading
+import time
+
+from opentelemetry.sdk.metrics.export import MetricsExportResult
+
+from azure_monitor.sdk.auto_collection import live_metrics
+from azure_monitor.sdk.auto_collection.live_metrics.exporter import (
+    LiveMetricsExporter,
+)
+from azure_monitor.sdk.auto_collection.live_metrics.sender import (
+    LiveMetricsSender,
+)
+
+FALLBACK_INTERVAL = 60
+PING_INTERVAL = 5
+POST_INTERVAL = 1
+MAIN_INTERVAL = 2
```

Comment:
What is this interval for? Is it only used until initial ping vs post state is determined?

Reply:
Is the manager thread interval, it will check if is ping or post action every 2 seconds

Reply:
It would be good to add some comments on what these are.